app/controllers/miq_policy_controller.rb

```diff
@@ -407,18 +503,11 @@ def replace_right_cell(options = {})
       right_cell_text += _(" (Names with \"%{search_text}\")") % {:search_text => @search_text} if @search_text.present? && !@folders
     when 'p'
       presenter.update(:main_div, r[:partial => 'policy_details', :locals => {:read_only => true}])
-      model_name = ui_lookup(:model => @sb[:nodeid].try(:camelize))
       if @policy.id.blank?
-        right_cell_text = if @sb[:mode]
-                            _("Adding a new %{model_name} %{mode} Policy") %
-                              {:model_name => model_name, :mode => _(@sb[:mode].capitalize)}
-                          else
-                            _("Adding a new %{model_name} Policy") % {:model_name => model_name}
-                          end
+        right_cell_text = adding
       else
-        options = {:model => "#{model_name} #{@sb[:mode] ? _(@sb[:mode].capitalize) : ""}",
-                   :name  => @policy.description}
-        right_cell_text = @edit ? _("Editing %{model} Policy \"%{name}\"") % options : _("%{model} Policy \"%{name}\"") % options
+        editing += N_(" \"%{name}\"") % {:name => @policy.description}
+        right_cell_text = _(editing)
```

Comment:
This won't work though. The concatenated `editing` string will not exist in the catalog.

Reply:
how about now?